README.md

```diff
@@ -118,9 +118,11 @@ task-02.02   2015-09-04   2eb21d8f9694146deca8c69cbc4a82acd62d395f   success
 ---------------
 
 Many of the specs have python script to run the tests automatically. By default
-experiments will be launched on [IoT-LAB saclay site] since it has most of the
-boards used in the release-specs. It can also be ran locally as long as the
-required `BOARDs` are available (if not ran on native).
+experiments will be launched on [IoT-LAB saclay site] that require non-`native`
+boards. The saclay site was chosen since it has most of the non-`native` boards
```

Comment:
Fixing my own mistakes ;-)

```suggestion
experiments will be launched on [IoT-LAB saclay site] for tests that require
non-`native` boards. The saclay site was chosen since it has most of the non-`native` boards
```

Needs adaptation for line-length